<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegGridMaskDialog</class>
 <widget class="QDialog" name="RegGridMaskDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Regular Grid Mask Selection</string>
  </property>
  <widget class="QLabel" name="label_regGridMaskProcImg">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>420</width>
     <height>420</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_regMaskInputs">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>70</y>
     <width>220</width>
     <height>220</height>
    </rect>
   </property>
   <property name="title">
    <string>Inputs</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="comboBox_regMask_ImgDim">
        <item>
         <property name="text">
          <string>2D Data</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3D Data</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4D Data</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_regMask_percentage">
          <property name="text">
           <string>Ratio</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_regMask_percentage"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_regMask_imgUpl">
        <property name="text">
         <string>Open Image Data</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_regMask_run">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_regMaskOrigImg">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>480</y>
     <width>390</width>
     <height>390</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>860</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_regmask_ImgName">
      <property name="text">
       <string>File Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_regMaskImgName"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_regMaskImgSave">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSlider" name="horizontalSlider_regGridMaskProcImg">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>530</y>
     <width>400</width>
     <height>17</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_regGridOrigImg">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>890</y>
     <width>370</width>
     <height>17</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
